src/client/Microsoft.Identity.Client/ApiConfig/AbstractAcquireTokenParameterBuilder.cs

```diff
@@ -423,6 +403,19 @@ internal void ValidateAndCalculateApiId()
             CommonParameters.ApiId = CalculateApiEventId();
             CommonParameters.ApiTelemId = ApiTelemetryId;
             CommonParameters.CorrelationId = CommonParameters.UseCorrelationIdFromUser ? CommonParameters.UserProvidedCorrelationId : Guid.NewGuid();
+
+#if DESKTOP || NET_CORE
+            if (CommonParameters.UsingProofOfPossesion && CommonParameters.PopAuthenticationConfiguration != null)
+            {
+                if (CommonParameters.PopAuthenticationConfiguration.PopCryptoProvider == null)
+                {
+                    CommonParameters.PopAuthenticationConfiguration.PopCryptoProvider = ServiceBundle.PlatformProxy.GetDefaultPoPCryptoProvider();
```

Comment:
Hmm, this object (the config) belongs to the user, so it would be better if MSAL didn't modify it. I'd keep the pop crypto provider object as a member of the auth scheme.